iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/serviceclient/JobClientCommon.java

```diff
@@ -93,10 +106,10 @@ private JobResult queryJobResponseResult(String jobId, JobType jobType, JobStatu
                 return jobResult;
             }
         }
-        throw new IotHubException("queryDeviceJob do not find the job");
+        throw new AssertionError("queryDeviceJob did not find the job");
     }
 
-    //@BeforeClass
```

Comment:
We have @BeforeClass in the actual test classes (JobClientIT and JobClientITonAndroid) and this setup is called from them.  Would this be triggered twice if we have it as @BeforeClass as well??

Reply:
Good point. We no longer need the @BeforeClass here since the @BeforeClass of the child calls into this method anyways.